<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device Manager Utility</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <property name="spacing">
     <number>12</number>
    </property>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="info_label">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>DEVICE MANAGER</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <widget class="QPushButton" name="refresh_button">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>refresh list</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="send_button">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>send</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="read_button">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>read</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="headerVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="headerCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="headerHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">3</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
